common/changes/@bentley/imodel-react-hooks/master_2022-02-28-07-26.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/imodel-react-hooks",
+      "comment": "#797363 added @bentley/telemetry-client to resolve module not found issue",
+      "type": "patch"
```

Comment:
I don't want to release a new ver of this PKG in 2.x since i have a PR to bump this to 3.0. Set the type to `"none"`

Reply:
updated to "none", Thanks